install.sh

```diff
@@ -108,20 +112,7 @@ install -d ${PREFIX}/share/dbus-1/services
 install -m 644 data/dbus/com.gexperts.Tilix.service ${PREFIX}/share/dbus-1/services/
 
 # Copy man page
-echo "Installing man pages"
-install -d ${PREFIX}/share/man/man1
-install -m 644 data/man/tilix ${PREFIX}/share/man/man1/tilix.1
-gzip -f ${PREFIX}/share/man/man1/tilix.1
-
-if type po4a-translate >/dev/null 2>&1; then
-    for f in data/man/po/*.man.po
-    do
-        LOCALE=$(basename "$f" .man.po)
-        install -d ${PREFIX}/share/man/${LOCALE}/man1
-        po4a-translate -k 0 -f man -m data/man/tilix -p data/man/po/${LOCALE}.man.po -l ${PREFIX}/share/man/${LOCALE}/man1/tilix.1
-        gzip -f ${PREFIX}/share/man/${LOCALE}/man1/tilix.1
-    done
-fi
+. "$(dirname $(realpath ${0}))/data/scripts/install-man-pages.sh"
```

Comment:
The quotes should be around `${0}` instead of `$()`. `$()` already prevents word splitting, it doesn't need quotes. `${0}` can include spaces so it does need to be quoted.